integration/simulation/transaction_injector.go

```diff
@@ -91,6 +96,12 @@ func NewTransactionInjector(
 // Deposits an initial balance in to each wallet
 // Generates and issues L1 and L2 transactions to the network
 func (m *TransactionInjector) Start() {
+	// always deploy it from the first wallet
+	// since it has a hardcoded key
+	m.deployERC20(m.issuingWallet)
+	// enough time to process everywhere
+	time.Sleep(m.avgBlockDuration * 6)
```

Comment:
What if this pooled the node for the transaction receipt instead of sleeping ?

Reply:
yeah, that's the next step, but it requires some work to expose receipts. In a future work.
But good point, I'll add a ticket to not forget about this
https://github.com/obscuronet/obscuro-internal/issues/479